### PR TITLE
VideoPress: add storage meter to the Admin page

### LINKED ISSUE #26474: VideoPress: add storage meter
Implements the storage meter in the UI.

Image

It will warn the user to upgrade to a higher plan when it achieves the storage limit.

### PR DESCRIPTION
**Pull request by retrofox**
This PR shows the storage meter in the UI:
* Pull the data from the site options
* Stores the value in the state
* Creates a new `<ConnectVideoStorageMeter />` component to print the values

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: 

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

* Go to VideoPress admin dashboard
* Confirm you see the storage meter

image

image

Disclaimer: we may need to improve the message when the space used is minimum.

Screen Shot 2022-09-27 at 10 05 51

### CODE REVIEW

File: projects/packages/videopress/src/client/admin/components/video-storage-meter/index.tsx [hunk 1]
```diff
@@ -38,9 +44,23 @@ const VideoStorageMeter: React.FC< VideoStorageMeterProps > = ( { className, tot
...
 
+export const ConnectVideoStorageMeter = props => {
+	const { storageUsed } = useVideos();
+	const total = 1024 * 1024 * 1024 * 1024;
+
+	if ( ! storageUsed ) {
+		return <VideoStorageMeter { ...props } used={ 0 } total={ 1 } />;
+	}
+
+	return <VideoStorageMeter { ...props } used={ storageUsed } total={ total } />;
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/admin/components/video-storage-meter/index.tsx]: Here I think we should check if the user has the `1TB` feature, as there is the unlimited feature as well.

**Comment by lhkowalski** [comment]
I was able to see the meter on the page, as expected:

videopress-storage-meter

**Review by dhasilva — DISMISSED** [review]
LGTM

**Review by dhasilva — DISMISSED** [review]

**Review by dhasilva — DISMISSED** [review]

**Review by dhasilva — APPROVED** [review]

**Comment by jsnmoon** [comment]
r253319-wpcom

**Pull request merged**